common/utils/EncryptionManager.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const crypto = require('crypto');
+const config = require('../config');
+const CONST = require('../constants');
+
+class EncryptionManager {
+
+  constructor(algorithm) {
+    this.algorithm = algorithm;
```

Comment:
should not be doing this as explained above. as we need to additionally validate etc.